dbaas-lib/engines/k8s-pxc/controller.go

```diff
@@ -249,3 +256,49 @@ func generatePass() ([]byte, error) {
 
 	return b, nil
 }
+
+func (p *PXC) checkClusterPods(name string, st *k8sStatus) error {
+	for i := 0; i < int(st.Status.PXC.Size); i++ {
+		podData, err := p.cmd.GetObject("pod", name+"-pxc-"+strconv.Itoa(i))
```

Comment:
What if in later versions of pxc/psmdb operators naming template would change?
Can we use labels instead?

Reply:
as I know, we dont have pod number in labels

Reply:
But we can retrieve a list of pods by label

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/101/commits/7ce73ab37cf608c44c28315bf025ea3a3746a2e1